doc/sphinx/source/conf.py

```diff
@@ -0,0 +1,38 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# For the full list of built-in configuration values, see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+import os
+import sys
+
+# -- Project information -----------------------------------------------------
+# https://www.sphinx-doc.org/en/master/usage/configuration.html#project-information
+
+project = "Neo4j Graph Data Science Python Client"
+copyright = "2023, Neo4j"
```

Comment:
is it `Neo4j Inc` nowadays?
Although our Java copyright header has `Neo4j` or `Neo4j Sweden AB` in it :shrug: